content/en/docs/Getting-started/tasks.md

```diff
@@ -20,6 +20,8 @@ This tutorial shows you how to
 1.  [Install minikube](https://minikube.sigs.k8s.io/docs/start/). You only have
     to complete the step 1, "Installation".
 
+1.  Ensure docker is installed and running. 
```

Comment:
Thanks for the PR @clintonsteiner.

This is already covered in the minikube install instructions linked above:

> What you’ll need
> …
> Container or virtual machine manager, such as: **[Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/)**, [QEMU](https://minikube.sigs.k8s.io/docs/drivers/qemu/), [Hyperkit](https://minikube.sigs.k8s.io/docs/drivers/hyperkit/), [Hyper-V](https://minikube.sigs.k8s.io/docs/drivers/hyperv/), [KVM](https://minikube.sigs.k8s.io/docs/drivers/kvm2/), [Parallels](https://minikube.sigs.k8s.io/docs/drivers/parallels/), [Podman](https://minikube.sigs.k8s.io/docs/drivers/podman/), [VirtualBox](https://minikube.sigs.k8s.io/docs/drivers/virtualbox/), or [VMware Fusion/Workstation](https://minikube.sigs.k8s.io/docs/drivers/vmware/)

Reply:
Ah - I didn't drill into the minikube instructions
I'm fine leaving it out then

Reply:
Rebased to just push the update to the logs